### PR TITLE
Cleanup static web interface assets.

### PR DESCRIPTION
**Pull request by dennisoelkers**
There were a lot of static js/css assets from ancient and dark times. This PR cleans up a lot (but not all) of them.

In case of unreferenced assets, they are delete straightaway.
In case of referenced static assets, they are deleted and references are replaced with a reference to a packaged version.

### CODE REVIEW

File: graylog2-web-interface/src/routing/AppFacade.jsx [hunk 1]
```diff
@@ -10,13 +10,12 @@ const SessionStore = StoreProvider.getStore('Session');
...
 
-import 'javascripts/shims/styles/shim.css';
-import 'stylesheets/bootstrap.min.css';
+import 'bootstrap/dist/css/bootstrap.css';
 import 'stylesheets/font-awesome.min.css';
 import 'stylesheets/newfonts.less';
 import 'stylesheets/bootstrap-submenus.less';
-import 'stylesheets/toastr.min.css';
-import 'stylesheets/rickshaw.min.css';
+import 'toastr/toastr.less';
```
Comment (joschi) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: Shouldn't this be [`toastr/build/toastr.min.css`](https://github.com/CodeSeven/toastr/blob/2.1.2/build/toastr.min.css)?

File: graylog2-web-interface/src/routing/AppFacade.jsx [hunk 2]
```diff
@@ -10,13 +10,12 @@ const SessionStore = StoreProvider.getStore('Session');
...
+import 'rickshaw/rickshaw.css';
```
Comment (joschi) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: Shouldn't this be [`rickshaw/rickshaw.min.css`](https://github.com/shutterstock/rickshaw/blob/v1.5.1/rickshaw.min.css)?

File: graylog2-web-interface/src/routing/AppFacade.jsx [hunk 3]
```diff
@@ -10,13 +10,12 @@ const SessionStore = StoreProvider.getStore('Session');
 const ServerAvailabilityStore = StoreProvider.getStore('ServerAvailability');
 const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
-import 'javascripts/shims/styles/shim.css';
-import 'stylesheets/bootstrap.min.css';
+import 'bootstrap/dist/css/bootstrap.css';
```
Comment (joschi) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: Shouldn't this be [`bootstrap/dist/css/bootstrap.min.css`](https://github.com/twbs/bootstrap/blob/v3.3.7/dist/css/bootstrap.min.css)?

Comment (dennisoelkers) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: Minification should be done by the toolchain, namely webpack.

File: graylog2-web-interface/src/routing/AppFacade.jsx (hunk 1 above)
Comment (dennisoelkers) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: Minification should be done by the toolchain, namely webpack.

File: graylog2-web-interface/src/routing/AppFacade.jsx (hunk 2 above)
Comment (dennisoelkers) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: Minification should be done by the toolchain, namely webpack.

File: graylog2-web-interface/src/routing/AppFacade.jsx (hunk 3 above)
Comment (joschi) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: ✅

File: graylog2-web-interface/src/routing/AppFacade.jsx (hunk 1 above)
Comment (joschi) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: ✅

File: graylog2-web-interface/src/routing/AppFacade.jsx (hunk 2 above)
Comment (joschi) [inline graylog2-web-interface/src/routing/AppFacade.jsx]: ✅

**Comment by joschi** [comment]
@dennisoelkers There are merge conflicts. Afters they've been resolved, this PR can be merged.

**Comment by dennisoelkers** [comment]
Rebased and resolved conflicts ✅

**Pull request merged**